src/cljs/vanilla/widgets/core.cljs

```diff
@@ -3,16 +3,19 @@
             [vanilla.widgets.basic-widget :as basic]
             [vanilla.widgets.make-chart :as mc]))
 
-(def widget-store (atom {}))
+;(def widget-store (atom {}))
 
 ; TODO - is having a widget registry even worth doing?
 
-(defn register-widget [name w]
-  (swap! widget-store assoc name w))
+;(defn register-widget [name w]
+;  (swap! widget-store assoc name w))
 
 ;(prn "register-widget " type
 ;  " //// w " w))
 
+;(defn build-simple [type data options]
```

Comment:
remove this dead code

Reply:
addressed all your comments, and changes are in repo, commit:
194b90df92d28746cc45764451ff5dfa87ea6ec8